deploy/docker-compose/server-bare.yml

```diff
@@ -11,8 +11,9 @@ services:
       MYSQL_ROOT_PASSWORD: "sota-test"
       MYSQL_USER: "sota"
       MYSQL_PASSWORD: "s0ta"
-      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_core sota_core_test"
-
+      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_resolver% sota_core sota_core_test sota_core% sota_device_registry"
+    command: ["--character-set-server=utf8", "--collation-server=utf8_unicode_ci", "--max-connections=1000"]
```

Comment:
I am not sure this is enough to run the random tests (see teamcity task) because the mariadb:stable tag is quite old and not sure it accepts these flags?